README.rst

```diff
@@ -1,7 +1,7 @@
 Django CKEditor
 ===============
 
-**NOTICE: django-ckeditor 5 has backwards incompatible code moves against 4.5.1.**
+**NOTICE: django-ckeditor 5 has backward incompatible code moves against 4.5.1.**
```

Comment:
The Django docs contain both variants, but the variant with an `s` at the end seems to be more common and also sounds better to me.

Reply:
Without `s` is American + world style. With `s` it's Britain only.

Reply:
Thanks, I didn't know this.